src/main/java/uk/gov/justice/digital/delius/jpa/standard/repository/ManagementTierRepository.java

```diff
@@ -7,4 +7,6 @@
 
 @Repository
 public interface ManagementTierRepository extends JpaRepository<ManagementTier, ManagementTierId> {
+
+    ManagementTier findFirstByIdOffenderIdOrderByIdDateChangedDesc(Long offenderId);
```

Comment:
Tried this and get:
```
Caused by: org.springframework.data.mapping.PropertyReferenceException: No property 'offenderId' found for type 'ManagementTier'
```
I believe the Id is needed because its an embedded ID and the offenderId and dateChanged fields are within the id field in the entity https://github.com/ministryofjustice/community-api/blob/main/src/main/java/uk/gov/justice/digital/delius/jpa/standard/entity/ManagementTier.java#L33

Reply:
Andy and I discussed this yesterday and reached the same conclusion